reporting-guide.md

```diff
@@ -1,18 +1,37 @@
 
-## Reports Endpoint User Guide
+---
+
+title: Reports User Guide
+seo-title: Analytics API Reports User Guide
+description: This user guide provides instructions and examples for using the reports endpoint for Adobe Analytics APIs.
+seo-description: 'Analytics APIs' 'Analytics reports' 'reports endpoint' 'adobe analytics' 'analytics reports api'
+
+---
+# Analytics API Reports User Guide
```

Comment:
Do you know if this header tag looks ok? It looks like we are using the ## in most places

Reply:
This is per markdown heading hierarchy. 

 This is per the "collaborative documentation model" instructions.

Reply:
If this is our way forward then we should update the other docs that don't do it this way for consistency.